src/cli_utils.py

```diff
@@ -5,6 +5,19 @@
 
 import re
 
+hash__node__group = {
```

Comment:
Why does this name use double underscores?

Reply:
Because they are for the `--group` and `--node` flags